drivers/gpio/gpio_esp32.c

```diff
@@ -8,7 +8,9 @@
 #define DT_DRV_COMPAT espressif_esp32_gpio
 
 /* Include esp-idf headers first to avoid redefining BIT() macro */
+#ifndef CONFIG_SOC_ESP32C3
 #include <soc/dport_reg.h>
```

Comment:
I guess thid #ifdef and `#include <soc/dport_reg.h>` can be replace by `#include <soc.h>`, as `dport_reg.h` is in there already. Can you check?

Reply:
Up to date.